bundles/org.openhab.binding.sunsynk/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,179 @@
+
+	<bridge-type id="account">
+		<label>SunSynk Account</label>
+		<description>Access to SunSynk Account. Used to discover plants and inverters tied to account.</description>
+		<properties>
+			<property name="vendor">SunSynk</property>
+		</properties>
+		<config-description>
+			<parameter name="email" type="text" required="true">
+				<label>E-mail Address</label>
+				<description>E-mail address for your SunSynk Connect account.</description>
+				<context>email</context>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<description>Password for your SunSynk Connect account.</description>
+				<context>password</context>
+			</parameter>
+		</config-description>
+
+	</bridge-type>
+
+	<thing-type id="inverter">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="account"/>
+		</supported-bridge-type-refs>
+		<label>SunSynk Inverter</label>
+		<description>Inverter thing for SunSynk Binding.</description>
+
+		<channels>
+			<channel id="interval-1-grid-charge" typeId="grid-charge"/>
+			<channel id="interval-1-grid-time" typeId="grid-time"/>
+			<channel id="interval-1-grid-capacity" typeId="battery-target"/>
+			<channel id="interval-1-grid-power-limit" typeId="power-charge-limit"/>
+			<channel id="interval-2-grid-charge" typeId="grid-charge"/>
+			<channel id="interval-2-grid-time" typeId="grid-time"/>
+			<channel id="interval-2-grid-capacity" typeId="battery-target"/>
+			<channel id="interval-2-grid-power-limit" typeId="power-charge-limit"/>
+			<channel id="interval-3-grid-charge" typeId="grid-charge"/>
+			<channel id="interval-3-grid-time" typeId="grid-time"/>
+			<channel id="interval-3-grid-capacity" typeId="battery-target"/>
+			<channel id="interval-3-grid-power-limit" typeId="power-charge-limit"/>
+			<channel id="interval-4-grid-charge" typeId="grid-charge"/>
+			<channel id="interval-4-grid-time" typeId="grid-time"/>
+			<channel id="interval-4-grid-capacity" typeId="battery-target"/>
+			<channel id="interval-4-grid-power-limit" typeId="power-charge-limit"/>
+			<channel id="interval-5-grid-charge" typeId="grid-charge"/>
+			<channel id="interval-5-grid-time" typeId="grid-time"/>
+			<channel id="interval-5-grid-capacity" typeId="battery-target"/>
+			<channel id="interval-5-grid-power-limit" typeId="power-charge-limit"/>
+			<channel id="interval-6-grid-charge" typeId="grid-charge"/>
+			<channel id="interval-6-grid-time" typeId="grid-time"/>
+			<channel id="interval-6-grid-capacity" typeId="battery-target"/>
+			<channel id="interval-6-grid-power-limit" typeId="power-charge-limit"/>
+			<channel id="interval-1-gen-charge" typeId="gen-charge"/>
+			<channel id="interval-2-gen-charge" typeId="gen-charge"/>
+			<channel id="interval-3-gen-charge" typeId="gen-charge"/>
+			<channel id="interval-4-gen-charge" typeId="gen-charge"/>
+			<channel id="interval-5-gen-charge" typeId="gen-charge"/>
+			<channel id="interval-6-gen-charge" typeId="gen-charge"/>
```

Comment:
Many typeId's are re-used, very good! Only thing is that the label is not overriden, so they will all look the same and users won't be able to distinguish them?
I would expect:
```
<channel id="interval-1-gen-charge" typeId="gen-charge">
    <label>Example Overriden Label</Label>
</channel>
```

Reply:
Done